index.js

```diff
@@ -0,0 +1,48 @@
+import Books from './modules/books.js';
+import dateTime from './modules/luxon.js';
+
+const listLink = document.querySelector('#list-link');
+const addNewLink = document.querySelector('#add-new-link');
+const contactLink = document.querySelector('#contact-link');
+const list = document.querySelector('#lists');
+const addNew = document.querySelector('#add-new');
+const contactUs = document.querySelector('#contact-us');
+
+const enableList = () => {
+  list.style.display = 'flex';
+  addNew.style.display = 'none';
+  contactUs.style.display = 'none';
+};
+
+const enableAddNew = () => {
+  list.style.display = 'none';
+  addNew.style.display = 'flex';
+  contactUs.style.display = 'none';
+};
+
+const enableContactUs = () => {
+  list.style.display = 'none';
+  addNew.style.display = 'none';
+  contactUs.style.display = 'flex';
+};
+
+listLink.addEventListener('click', enableList);
+addNewLink.addEventListener('click', enableAddNew);
+contactLink.addEventListener('click', enableContactUs);
+
+// Date time
+document.querySelector('#date').innerHTML = dateTime;
+const a = new Books();
+a.createObject();
+a.displayBooks();
+a.saveToLocalStorage();
+
+const form = document.getElementById('add-new-form');
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  a.addBooks();
+});
+
+window.removeBook = (id) => {
+  a.removeBooks(id);
+};
```

Comment:
- [ ] Good job so Far, :clap: , However on page load all your contents show on the browser at once without clicking on any navigation buttons, which is not supposed to happen, below is the screen of the browser on page load: :arrow_down: 

![OnPageLoad](https://github.com/HossainAraf/Awesome-Books/assets/43528594/f7a07857-e3a7-4fd0-b768-4f2ad7565ecd)


- [ ] Possible solution could be to use the below line of code at the bottom of your index.js file and adjust your alignments accordingly: :arrow_down: 
`window.addEventListener('DOMContentLoaded', () => {`
  `const sections = [list, addNew, contactUs];`
  `sections.forEach((s) => {`
    `s.style.display = 'none';`
 ` });`
`});`